src/components/OrganizationCard/OrganizationCard.spec.tsx

```
@@ -347,24 +348,157 @@
+      const cancelMutation = successMocks.find(
+        mock => mock.request.query === CANCEL_MEMBERSHIP_REQUEST
+      );
+      expect(cancelMutation?.result).not.toHaveBeenCalled;
+    });
 
+  
+
+  it('should handle withdrawal attempt with no userId', async () => {
+    // Mock getItem to return null to simulate no userId
+    mockGetItem.mockReturnValue(null);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)